config/locales/en.yml

```diff
@@ -3630,6 +3630,11 @@ See the %{link} to find out more about %{sitename}'s features and to start using
         paid_order: "Your order was paid so %{distributor} has refunded the full amount"
         credit_order: "Your order was paid so your account has been credited"
         subject: "Cancellation of Order"
+      cancel_email_for_shop:
+        greeting: "Dear %{name},"
+        subject: "Cancellation of Order"
+        intro: "An order has just been cancelled."
```

Comment:
So this is only happening when an order is cancelled by a customer, right? I feel like we could add a bit more to the message here, like `A customer has cancelled their order` or something which conveys that context a bit.